lib/condition/condition.ts

```diff
@@ -28,13 +28,13 @@ export function empty_condition(): Condition {
 export function parse_condition(source: string, baseIRI: string): Condition {
     const shapeQuads = new Parser().parse(SHAPES);
     const output = extractShapes(shapeQuads, {
-        "https://w3id.org/tree#And": (obj) =>
+        "http://vocab.deri.ie/csp#And": (obj) =>
```

Comment:
Wanted to check out the vocabulary, but cannot dereference it :(

Reply:
it's this boy https://lov.linkeddata.es/dataset/lov/vocabs/csp

Reply:
Yeah, I also found https://lov.linkeddata.es/dataset/lov/terms/?&type=class&tag=Industry&vocab=csp listing the different URIs, but still sad that it doesn't dereference.